kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md

```diff
@@ -244,154 +244,151 @@ It is no doubt true that in most vratas brāhmaṇas are to be fed, but one shou
+The keynote struck by the Matsya, Brahma and other Purāṇas is how to secure great rewards with little effort. The Brahma-purāṇa remarks that by merely taking in Kaliyuga the name of Keśava a man obtains the same rewards that one got in kṛta-yuga by deep meditation, in Tretā by performing sacrifices and in Dvāpara by worship (of images). The Matsya says: The great sages do not commend sacrifices which involve killing (of animals); those whose wealth is _tapas_ became firmly placed in heaven by making gifts, according to their means, of particles of grains (collected from fields &c.), roots, fruits, vegetables and water vessels and that _tapas_ is superior for (several) reasons to sacrifices. The Padmapurāṇa reaches the peak of exaggeration when it says 'no one except Hari will be able to recount the merit of him who offers a lamp on the day of Hari in Kārtika with devotion' or 'Viṣṇuvrata is the highest, a hundred solemn Vedic sacrifices are not equal to it'. The Skandapurāṇa[^112] (quoted by Hemādri on vrata vol. I. pp. 318-321 ) contains a long eulogy of vrata, one verse of which may be cited here 'Gods attained their position through restrictive observances (vratas); they shine forth as stars by virtue of the vratas ( they performed)'.
 
 There were certain differences between Vedic sacrifices and vratas. The promised reward of most sacrifices was heaven, while Purāṇas promised that most vratas would confer tangible benefits in this very life. Further, vratas could be performed by: all, including sūdras, maidens, married women, widows, even prostitutes.
 
-There were vratas for the brahmaçõrin (called Veda-vratas ) and for the spātaka. These have been already dealt with in H. of Dh. vol. II, pp. 370-374 and pp. 412-415 respectively. That
+There were vratas for the _brahmaçõrin_ (called Veda-vratas ) and for the sṇātaka. These have been already dealt with in H. of Dh. vol. II, pp. 370-374 and pp. 412-415 respectively.
```

Comment:
brahmacārin , snātaka

Reply:
corrected.

Reply:
There were vratas for the _brahmcārin_ (called Veda-vratas ) and for the snātaka. These have been already dealt with in H. of Dh. vol. II, pp. 370-374 and pp. 412-415 respectively.